docs/reference/upgrade.asciidoc

```diff
@@ -9,14 +9,20 @@ process so upgrading does not interrupt service. Rolling upgrades are supported:
 * Between minor versions
 * From 5.6 to 6.8
 * From 6.8 to {version}
+* Between {minor-version} versions
 
 
 The following table shows the recommended upgrade paths to {version}.
 
-[cols="<1m,3",options="header",]
+[cols="<1,3",options="header",]
 |====
 |Upgrade from   
-|Recommended upgrade path  to {version}
+|Recommended upgrade path to {version}
+
+ifeval::[ "{version}" != "{minor-version}.0" ]
+|A previous {minor-version} version (e.g., {minor-version}.0)
+|<<rolling-upgrades,Rolling upgrade>> to {version}
+endif::[]
 
 |7.0–7.5
```

Comment:
Similarly, instead of `7.5` is there a macro for the previous minor version?

Reply:
See https://github.com/elastic/elasticsearch/pull/55012#discussion_r406354701